packages/usa-step-indicator/src/usa-step-indicator.twig

```diff
@@ -1,4 +1,4 @@
-<div class="usa-step-indicator {{ modifier }}" aria-label="progress">
```

Comment:
Adding an `aria-label` to a `div` without a `role` is flagged by axe as an item that needs review:

![image](https://user-images.githubusercontent.com/93996430/198137362-d7deb359-c7a4-495b-903c-dc1908a317ae.png)

pa11y 3 (falsely) flags this as an error. I converted this to a `section` so that it inherits a role.